run.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+pip3 install -r requirements.txt
+cp application.py app.py
```

Comment:
Mmm what's the meaning of that?

Reply:
on the CS50IDE the lazy loading of flask would as a default load application.py, it seems that on the ubuntu machines on virtual box it has to be app.py so i am simply renaming the file...

Reply:
do you think i should just rename it on the repository?